financial-connections/src/main/java/com/stripe/android/financialconnections/domain/SaveAccountToLink.kt

```diff
@@ -62,16 +65,23 @@ internal class SaveAccountToLink @Inject constructor(
 
     private suspend fun ensureReadyAccounts(
         shouldPollAccountNumbers: Boolean,
-        partnerAccounts: List<CachedPartnerAccount>,
-        action: suspend (Set<String>) -> FinancialConnectionsSessionManifest,
+        partnerAccounts: List<CachedPartnerAccount>?,
+        action: suspend (Set<String>?) -> FinancialConnectionsSessionManifest,
     ): FinancialConnectionsSessionManifest {
-        val selectedAccountIds = partnerAccounts.map { it.id }.toSet()
-        val linkedAccountIds = partnerAccounts.mapNotNull { it.linkedAccountId }.toSet()
+        val selectedAccountIds = partnerAccounts?.map { it.id }?.toSet() ?: emptySet()
```

Comment:
This will result in a `0 accounts saved to Link` message. Are you planning to fix this later?

Reply:
actually good catch! covered it + tests here: https://github.com/stripe/stripe-android/pull/8727/commits/0644b5d257751290bc0ee91f3a93dca4f07a3e0a